### DISCUSSION TITLE
Is EasyEngine still being developed?

### DISCUSSION THREAD

**Discussion by jauhari** [EasyEngine/easyengine#1814] [Polls]
I am curies why there are no update does This Awesome Project still being developed?

**Comment by prionkor** [comment]
looking at the last commit, I would say no. Its sad such an awesome project has became stale. sad reality of open source!

**Reply by jauhari** [reply]
does there any **Good Alternative** for this one?

**Comment by marcoandrei-waycomm** [comment]
Hello! The last commit was in the last August, I'm pretty sure that the guys are working on this project actively.

**Comment by jauhari** [comment]
I hope there any any clues how to make Custom Port on EasyEngine server. Example multiple domain with different port each on it
